src/middlewared/middlewared/plugins/pool.py

```diff
@@ -725,15 +725,6 @@ class Config:
     async def pool_scrub_extend(self, data):
         data['pool'] = data.pop('volume')
         data['pool'] = data['pool']['id']
-        if data['dayweek'] == '*':
-            data['dayweek'] = [str(value) for value in range(1, 8)]
-        else:
-            data['dayweek'] = data['dayweek'].split(',')
-
-        if data['month'] == '*':
-            data['month'] = [str(value) for value in range(1, 13)]
-        else:
-            data['month'] = data['month'].split(',')
         return data
 
     async def validate_data(self, data, schema):
```

Comment:
Forgot in first review: either add @private or make it a python private method `__validate_data`